Classes/Provider/ContentProvider.php

```diff
@@ -198,4 +198,21 @@ public function getPriority(array $row) {
 		return 0;
 	}
 
+	/**
+	 * Get preview chunks - header and content - as
+	 * array(string $headerContent, string $previewContent, boolean $continueRendering)
+	 *
+	 * @param array $row The record data to be analysed for variables to use in a rendered preview
+	 * @return array
+	 */
+	public function getPreview(array $row) {
+
+		if ($this->contentObjectType !== $row['CType']) {
+			return NULL;
```

Comment:
must `return array(NULL, NULL, TRUE)` (sorry for the esoteric format...). I think we're casting this value, but `return NULL` would cause conflict with method signature.